6_3_C.py

```diff
@@ -0,0 +1,31 @@
+def get_letters():
+    """
+    :return: List that includes all the letters
+    """
+    small_letter = [chr(i) for i in range(ord('a'), ord('z') + 1)]
+    capital_letter = [chr(i) for i in range(ord('A'), ord('Z') + 1)]
+    return small_letter + capital_letter + [" "]
+
+
+def count_words(text):

```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/